app/controllers/application_controller/network.rb

```diff
@@ -4,6 +4,7 @@ module ApplicationController::Network
   def network_managers
     openstack_network_manager = Rbac::Filterer.filtered(ManageIQ::Providers::Openstack::NetworkManager).select(:id, :name, :parent_ems_id)
     redhat_network_manager = Rbac::Filterer.filtered(ManageIQ::Providers::Redhat::NetworkManager).select(:id, :name, :parent_ems_id)
+    openstack_network_manager = openstack_network_manager.select { |manager| EmsCloud.find_by_id(manager.parent_ems_id) }
```

Comment:
I am think you can change input query for RBAC of openstack_network_manager
```ruby
ManageIQ::Providers::Openstack::NetworkManager.joins(:parent_manager).where(:parent_managers_ext_management_systems=>{:type=>'ManageIQ::Providers::Openstack::CloudManager'})
```

Reply:
Done :) Thanks for review.